src/sider/aktivitetskrav/vurdering/SendForhandsvarselSkjema.tsx

```diff
@@ -20,6 +20,8 @@ import { Brevmal } from "@/data/aktivitetskrav/forhandsvarselTexts";
 const texts = {
   title: "Send forhåndsvarsel",
   beskrivelseLabel: "Begrunnelse (obligatorisk)",
+  begrunnelseDescription:
+    "Begrunnelsen din blir en del av en større brevmal. Se forhåndsvisning for å se hele varselet",
```

Comment:
`Klikk på forhåndsvisning for å se hele varselet.` eller `Åpne forhåndsvisning for å se hele varselet.` 🤷🏼‍♂️
Bare så det ikke ble dobbelt "se". Og så punktum på slutten.

Reply:
Ja lurt!